### ISSUE TITLE
Other ostatus compatible servers

### ISSUE THREAD

**Issue by Shnoulle** [ialisme/fed.ialis.me#12]
What about other compatible server ?

For example [Friendica](http://friendi.ca/) , but GnuSocial fork and friendica fork too ? 

Are Qvitter must have own section or not ? For user : there are a lot of difference between GS and qvitter.

**Comment by hrefhref** [comment]
For sure, they should be added too :) I'll create their entries/pages.

I didn't new about Qvitter but if it has differences, it needs to be in the list too.

The servers not yet stable will be added to (like Plemora)

**Comment by Shnoulle** [comment]
Great, i look for other GS fork.

For qvitter : maybe https://quitter.se for link (`https://quitter.se/hannes2peer` is the lead dev of qvitter and admin of quitter.se). I put some doc for user when i found time ;)

**Comment by hrefhref** [comment]
thanks :green_heart:

**Comment by lanodan** [comment]
From memory Quitter is a theme/plugin for gnusocial.
PostActive is listed as a fork of GS here: https://en.wikipedia.org/wiki/OStatus

**Comment by Shnoulle** [comment]
Yep, but qvitter update whole GUI for user (see https://quitter.se/main/public and https://status.fsf.org/main/public) 
And PostActive : repo https://git.postactiv.com/postActiv/postActiv . I never test it but seems some old GS user make pull request and reports. Seems you can use Qvitter on postactiv, unsure for TwitterBridge (i'm maybe one of the last one using it)

**Comment by hrefhref** [comment]
Ok so: qvitter is just a plugin, right ? So maybe it should be mentionned in the GS page or in the tools for GS (yet to be implented).

PostActiv: i'm gonna add this one.

Re-opening the issue :)

**Comment by hrefhref** [comment]
Also: Pleroma https://gitgud.io/lambadalambda/pleroma (still in development)

**Comment by hrefhref** [comment]
And https://github.com/deoxxa/don

**Comment by Shnoulle** [comment]
https://project.hubzilla.org ? 

fediverse include Diaspora , but Diaspora aren't Ostatus compatible (currently)

PS : seems hubzilla is near Friendica , but : seemsonly. Not same , really not same (but compatible)
